common/static/js/vendor/tinymce/BUILD_README.md

```diff
@@ -0,0 +1,53 @@
+# Instructions for updating tinymce to a newer version:
+
+1. Download the desired version from https://github.com/tinymce/tinymce/tags
+2. If it’s a major update, follow the official migration doc and update the codebase as needed.
+3. Setup the codemirror-plugin as per the instruction below.
+4. Find all the EDX specific changes in the currently used version of tinymce by searching for the string "EDX" in the vendor/js/tinymce dir.
+5. Merge the EDX specific changes with the newly downloaded version.
+6. Follow the instructions given below to create the new version of js/tinymce.full.min.js
+
+# Instruction for setting codemirror-plugin
+
+1. Download the tinymce-codemirror-plugin from https://gitlab.com/tinymce-plugins/tinymce-codemirror
+2. Open terminal in the downloaded plugin directory and run the following commands:
+    ```
+     npm install
+     npm run prepublish (This command will generate the minified file in the plugin directory)
+    ```
+3. Remove the tinymce-codemirror/plugins/codemirror/codemirror-4.8 directory
+4. Move the tinymce-codemirror/plugins directory to `common/static/js/vendor/tinymce/js/plugins/` directory.
+5. Apply EDX specific changes in the existing code to the `plugin.js` and `source.html` files.
+6. Install [uglify-js](https://www.npmjs.com/package/uglify-js) and generate `plugin.min.js`
+    ```
+    cd common/static/js/vendor/tinymce/js/plugins/codemirror/
+    uglify plugin.js -m -o plugin.min.js
```

Comment:
Why do we need to regenerate the plugin? Isn't it possible to to just drop the code into the `plugin` folder?

Reply:
@leangseu-edx, as described in the previous point, `common/static/js/vendor/tinymce/js/tinymce/plugins/codemirror/plugin.js` has custom changes (code drift). We've already ported them to TinyMCE v5, as you can see [here](https://github.com/openedx/edx-platform/pull/30335/files/9f139136c85961b2afb1a287bb26828ab19eb773#diff-83ed1677d3c5797ca90adcc2bb891d16249cec81d187e96b94984f5dec67a73bR23).